intezer_sdk/alerts_results.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+from typing import Dict
+from typing import Tuple
+
+from intezer_sdk.api import IntezerApiClient
+from intezer_sdk.api import raise_for_status
+from intezer_sdk.history_results import HistoryResult
+
+
+class AlertsHistoryResult(HistoryResult):
+
+    def __init__(self, request_url_path: str, api: IntezerApiClient, filters: Dict):
+        """
+        Fetch all alerts history results from server.
+        """
+        super().__init__(request_url_path, api, filters)
+
+    def _fetch_history(self, url_path: str, data: Dict
+                       ) -> Tuple[int, List]:
```

Comment:
Is this how the line should break? Do we use Black here?

Reply:
No :\ will fix it (again)